app/controllers/posts_controller.rb

```diff
@@ -1,5 +1,11 @@
 class PostsController < ApplicationController
-  def index; end
+  def index
+    @user = User.find_by(id: params[:user_id])
+    @posts = Post.where(author_id: params[:user_id])
+  end
```

Comment:
* Based on the wireframe provided your post-controller index page is supposed to render 5 most recent comments as shown in the wireframe below 👇 **HINT** 💭  you can create a method that will fetch the 5 most recent comments
![image](https://user-images.githubusercontent.com/52497246/214909894-8a614e3f-8219-46f7-8814-791448e98be1.png)